content/en/developers/dogstatsd/dogstatsd_mapper.md

```diff
@@ -0,0 +1,129 @@
+---
+title: DogStatsD Mapper
+kind: documentation
+description: Convert parts of statsd metric names to tags using mapping rules in DogStatsD.
+further_reading:
+    - link: 'developers/dogstatsd'
+      tag: 'Documentation'
+      text: 'Introduction to DogStatsD'
+    - link: 'developers/libraries'
+      tag: 'Documentation'
+      text: 'Official and Community created API and DogStatsD client libraries'
+    - link: 'https://github.com/DataDog/datadog-agent/tree/master/pkg/dogstatsd'
+      tag: 'GitHub'
+      text: 'DogStatsD source code'
+---
+
+With Agent v7.17+, the DogStatsD Mapper feature allows you to convert parts of a metric name submitted to DogStatsD to tags using mapping rules with wildcard and regex patterns. For example it allows you to transform the metric:
+
+- `airflow.job.duration.<JOB_TYPE>.<JOB_NAME>`
+
+into the metric `airflow.job.duration` with two associated tags:
+
+- `job_type:<JOB_TYPE>`
+- `job_name:<JOB_NAME>`.
+
+To create a mapping rule:
+
+1. [Open your `datadog.yaml` file][1].
+2. Add a [mapping rule configuration block](#mapping-rule-configuration) under the `dogstatsd_mapper_profiles` parameter.
+
+## Mapping rule configuration
+
+A mapping rule block have the following layout:
+
+```yaml
+dogstatsd_mapper_profiles:
+    - name: '<PROFILE_NAME>'
+      prefix: '<PROFILE_PREFIX>'
+      mappings:
+          - match: '<METRIC_TO_MATCH>'
+            match_type: '<MATCH_TYPE>'
+            name: '<MAPPED_METRIC_NAME>'
+            tags:
+                '<TAG_KEY>': '<TAG_VALUE_TO_EXPAND>'
+```
+
+With the following placeholders:
```

Comment:
I think it might worth indicating for all fields what's required and what's optional. See https://github.com/DataDog/datadog-agent/pull/4471/files#diff-99e505a5e29849d8a342596285e284a1R743-R753

Reply:
done